proteus/tests/test_gauges.py

```diff
@@ -14,9 +14,9 @@
 from proteus import default_p as p
```

Comment:
Since you've already accumulated quite a few tests, you should probably merge in master so shippable will start working for you again.